spyder_kernels/customize/spyderpdb.py

```diff
@@ -300,36 +308,73 @@ def interaction(self, frame, traceback):
             return super(SpyderPdb, self).interaction(
                 frame, traceback)
 
-    def print_stack_entry(self, frame_lineno, prompt_prefix='\n-> ',
-                          context=None):
+    def print_stack_entry(self, *args, **kwargs):
         """Disable printing stack entry if requested."""
         if self._disable_next_stack_entry:
             self._disable_next_stack_entry = False
             return
-        return super(SpyderPdb, self).print_stack_entry(
-            frame_lineno, prompt_prefix, context)
+        return super(SpyderPdb, self).print_stack_entry(*args, **kwargs)
```

Comment:
```suggestion
        return super().print_stack_entry(*args, **kwargs)
```

This is now valid in Python 3.